src/ex5/dist/main.js

```diff
@@ -0,0 +1,57 @@
+class Main {
+    constructor() {
+        this.itemClient = new ItemClient()
+    }
+
+    init = async () => {
+        const addItemButton = document.getElementById("list-item-submit");
+        addItemButton.addEventListener("click", this.handleItem);
+
+        await this.renderItems();
+    }
+
+    handleItem = async () => {
+        const input = document.getElementById("list-item-input");
+        const inputValue = input.value;
+
+        await this.itemClient.postItem(inputValue)
+        await this.renderItems()
+    }
+
+    deleteItem = async item => {
+        await this.itemClient.deleteItem(item);
+        await this.renderItems();
+    }
+
+    renderItems = async () => {
+        const list = document.getElementById("list");
+        list.innerHTML = "";
```

Comment:
I would add a function called `resetItemsList` that just runs this line to make you code look more like a "story"